src/components/Home/Filter/SkillsFilter.jsx

```diff
@@ -0,0 +1,34 @@
+const SkillsFilter = ({ setFilterValue }) => {
+    
+    
+    return (  
+        <>
+        <section className="skillsFilter">
+            <select className="filter_selection" 
+                    defaultValue="select-skill" 
+                    onChange={(event) => setFilterValue(event.target.value)} 
+            >
+                <option value={""}> All </option>
+                <option value={"Java"}> Java </option>
```

Comment:
With this try and use a list and a for loop/mapping. That way you don't have to type everything out manually.